releases/release-6.1.1.md

```diff
@@ -0,0 +1,121 @@
+---
+title: TiDB 6.1.1 Release Notes
+---
+
+# TiDB 6.1.1 Release Notes
+
+Release date: 2022-xx-xx
+
+TiDB version: 6.1.1
+
+## Compatibility changes
+
++ TiDB
+
+    (dup: release-6.2.0.md > Bug fixes> TiDB)- Fix the issue that `SHOW DATABASES LIKE …` is case-sensitive [#34766](https://github.com/pingcap/tidb/issues/34766)
+
+## Improvements
+
++ TiDB
+
+<!-- <planner> -->
+(dup: release-6.2.0.md > # Performance)[User document](/optimizer-hints.md#semi_join_rewrite) [#35323](https://github.com/pingcap/tidb/issues/35323)
+(dup: release-5.2.4.md > Bug fixes> TiDB)- Fix the issue that partitioned tables cannot fully use indexes to scan data in some cases [#33966](https://github.com/pingcap/tidb/issues/33966)
+
+<!-- <transaction> -->
+(dup: release-6.2.0.md > Bug fixes> TiDB)- Avoid sending requests to unhealthy TiKV nodes to improve availability [#34906](https://github.com/pingcap/tidb/issues/34906)
+
++ TiKV
+
+    - In the new backup organization structure, we will see: `./br backup --pd "127.0.0.1:2379" -s "s3://backup/20220621" - After br command finished, we will have the structure below. ➜ backup tree . . └── 20220621 ├── backupmeta ├── store1 │ └── backup-xxx.sst ├── store100 │ └── backup-yyy.sst ├── store2 │ └── backup-zzz.sst ├── store3 ├── store4 └── store5` [#13063](https://github.com/tikv/tikv/issues/13063)
+    (dup: release-6.2.0.md > Improvements> TiKV)- Support compressing the metrics response using gzip to reduce the HTTP body size [#12355](https://github.com/tikv/tikv/issues/12355)
+
+## Bug fixes
+
++ TiDB
+
+<!-- <execution> -->
+- executor: fix index_lookup_hash_join hang when used with limit [#35638](https://github.com/pingcap/tidb/issues/35638)
```

Comment:
Fix the issue that IndexLookupHashJoin may hangs when used with limit [#35638](https://github.com/pingcap/tidb/issues/35638)
Fix the issue that TiDB may panic during update stmt [#32311](https://github.com/pingcap/tidb/issues/32311)
Fix the bug that `show columns` may send cop request [#36496](https://github.com/pingcap/tidb/issues/36496)
Fix bug that `show warnings` may return `invalid memory address or nil pointer dereference` error [#31569](https://github.com/pingcap/tidb/issues/31569)
Fix the case sensitive issues for `show database like` statement [#34766](https://github.com/pingcap/tidb/issues/34766)
Fix bug that static partition prune may return wrong result for agg query if the table is empty [#35295](https://github.com/pingcap/tidb/issues/35295)